docs/docs/tutorial/part-1/index.mdx

```diff
@@ -0,0 +1,299 @@
+
+First, you write the code for your Gatsby site from your computer. When you're ready to make your changes live on the internet, you go through the following steps:
+
+1. You push your changes from your computer to a remote repository on GitHub. GitHub is an online platform for storing code for your projects.
+1. Gatsby Cloud watches your GitHub repository for changes. When it sees your new changes, Gatsby Cloud builds your site from your code on GitHub.
+1. Gatsby Cloud hosts the finished version of your site at a unique URL, which users can use to access the latest version of your site.
+
+</details>
+
+
+## Create a Gatsby site
+
+To create your first Gatsby site, you're going to use a command from the Gatsby command line interface (CLI): `gatsby new`. This command brings up an interactive prompt that asks you questions about the site you want to build. After you enter all the information, the CLI uses your answers to automatically generate your new Gatsby site.
+
+![The welcome message for the interactive "gatsby new" command.](./gatsby-new-cli.png)
+
+Let's take a closer look at the process:
+
+1. Open the command line, and use the `cd` command to change directories into the folder where you want to create your new Gatsby site. For example, if you wanted to create your new site on your desktop, you might type:
+```shell:title=~
+cd Desktop
+```
+
+2. Run the following command from the command line. This will start up the interactive prompt to help you create a new Gatsby site.
+```shell:title=~/Desktop
+gatsby new
+```
+
+3. When the prompt asks, **"What would you like to call your site?"** enter a name for your site.
+```shell
+What would you like to call your site?
+✔ · My First Gatsby Site
+```
+
+4. When the prompt asks, **"What would you like to name the folder where your site will be created?"** use the default folder name, which will be based on the site name you chose.
+```shell
+What would you like to name the folder where your site will be created?
+✔ Desktop/ my-first-gatsby-site
+```
+
+5. When the prompt asks, **"Will you be using a CMS?"** select **"No (or I'll add it later)"**.
+```shell
+✔ Will you be using a CMS?
+· No (or I'll add it later)
+```
+
+<Announcement>
+
+In the future, you can use these options to tell `gatsby new` what features you want to add to your site, and `gatsby new` will automatically configure them for you. It's a much quicker way to set up new projects.
+
+But in this first site, you'll set things up manually to learn about how Gatsby's pieces fit together.
+
+</Announcement>
+
+6. When the prompt asks, **"Would you like to install a styling system?"** select **"No (or I'll add it later)"**. (You'll add styles manually later.)
+```shell
+✔ Would you like to install a styling system?
+· No (or I'll add it later)
+```
+
+7. When the prompt asks, **"Would you like to install additional features with other plugins?"** select **"Build and host for free on Gatsby Cloud"**, then select **"Done"**.
```

Comment:
Compared to the previous prompts people will need to use the <kbd>Spacebar</kbd> to toggle different options. We should call that out, not that people navigate to an option an directly press <kbd>Enter</kbd>